### DISCUSSION TITLE
Does pulsar delay message support rebooking?

### DISCUSSION THREAD

**Discussion by liuhuagui** [apache/pulsar#18317] [Q&A] [answered]
For example, the previous delay time was changed from 5m to 30m or even 0, even deleted.

 If not, is there a plan?

**Comment by liuhuagui** [comment, marked as answer]
@coderzc Please help to answer🙏

**Reply by coderzc** [reply]
Pulsar does not support changing the scheduled time of the message that has been sent. No plan for the time being.

**Reply by liuhuagui** [reply]
Thanks！
